tests/system/test_cluster/test_agent_groups/test_agent_never_connected_change_groups.py

```diff
@@ -0,0 +1,127 @@
+"""
+copyright: Copyright (C) 2015-2022, Wazuh Inc.
+           Created by Wazuh, Inc. <info@wazuh.com>.
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+type: system
+brief: Check that when an agent with status never_connected, pointing to a master/worker node is
+       registered using agent-auth and when it is assigned to a group with agent-group, the change is 
+       synced in the cluster.
+tier: 1
+modules:
+    - cluster
+components:
+    - manager
+    - agent
+path: /tests/system/test_cluster/test_agent_groups/test_agent_never_connected_change_groups.py
```

Comment:
Change the module name and the path with the new module name.  Ex: test_agent_never_connected_added_to_group_by_tool.py